### ISSUE TITLE
Add Command that directly replaces or inserts the results of a Prompt at the cursor

### ISSUE THREAD

**Issue by MaxMoldmann** [logancyang/obsidian-copilot#468]
Love this Plugin!
One major improvement would be if I could bypass the chat window if I know I want to copy the result anyway into the note.

**Comment by logancyang** [comment]
Thanks for the suggestion @MaxMoldmann, basically an inline completion mode for chat right?

**Comment by MaxMoldmann** [comment]
Yes, you have implemented this already in the overlay chat window. The sidebar chat also has learned to insert text at cursor which is great.
**The last missing piece is "replace at cursor" instead of "insert at cursor".**
However instead of adding another button, you could change the behavior of insert at cursor to remove currently selected text - then it effectively becomes an "replace at cursor" as well.
